src/components/ebay-breadcrumb/browser.json

```diff
@@ -0,0 +1,6 @@
+{
+    "dependencies": [
+        "@ebay/skin/breadcrumb",
```

Comment:
I think you'll need to depend on the `utility` to use the `.clipped` class.
https://github.com/eBay/ebayui-core/blob/b9f3790c3811ed85629b64126196074d02cc7c4a/src/components/ebay-carousel/browser.json#L3

Please also add a `breadcrumb` link to the main `README.md` section:
https://github.com/eBay/ebayui-core/blob/0.2.0/README.md#components

Reply:
done